pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -12,7 +12,7 @@ ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.c
 ARG TORCHVISION_VERSION=0.14.1+cu117
 ARG TORCHAUDIO_VERSION=0.13.1+cu117
 ARG TORCHDATA_VERSION=0.5.1
-ARG TORCHSERVE_VERSION=0.7.1
+ARG TORCHSERVE_VERSION
 ARG SM_TOOLKIT_VERSION=2.0.14
```

Comment:
toolkit version can be removed here, right?

Reply:
Right. removed.